docs/source/table.rst

```diff
@@ -53,6 +53,19 @@
      - .. image:: https://licensebuttons.net/l/by-sa/4.0/80x15.png
           :target: https://creativecommons.org/licenses/by-sa/4.0
 
+    * - Da-tacos
```

Comment:
Can we change the name of the dataset to "Da-TACOS" in the docs?

Reply:
Sincerely, I am not sure. For consistency, if it can be used in the initialize module, it can be the name in all the docstrings. But I think that now from 3.0.0 release it is possible. 

@magdalenafuentes it is possible:

```Python
def __init__(self, data_home=None):
        super().__init__(
            data_home,
            name="da_tacos",
            track_class=Track,
            bibtex=BIBTEX,
            remotes=REMOTES,
            license_info=LICENSE_INFO,
        )
```
If I change da_tacos by Da-TACOS.... initialize would be with Da-Tacos and I could change da_tacos to Da-TACOS in all the documentation. Am I right? Do you see some inconvenient?

Reply:
@PRamoneda in the code, you're right we should use the lowercase `da_tacos`, but as Furkan suggested, when referencing it in the table here and in the dataset description, use the official name `Da-TACOS`!

Reply:
Hi @rabitt 
and here??
```Python
name="da_tacos",
```
Could I call it Da-TACOS???

If I change that to Da_TACOS for the final user everything is Da-TACOS.

Reply:
> @PRamoneda quick FYI - I made a small commit to update the structure of the Dataset class to reflect the new API in #443

Hi @rabitt ,

How can I access to full _metadata from the Track datacos class?

Reply:
@PRamoneda No, in the init, `name=da_tacos` needs to stay this way - that's part of the code :) But it can change in the docstrings -

For example like this line:
```
 * - Da-TACOS
 * - - chromagrams (crema, hpcp, cens)
      - madmom features
      - automatic extracted keys
```

And at the top of da_tacos.py, instead of:
```
"""da-tacos Dataset Loader
```
it should be
```
"""Da-TACOS Dataset Loader
```

Reply:
> > @PRamoneda quick FYI - I made a small commit to update the structure of the Dataset class to reflect the new API in #443
> 
> Hi @rabitt ,
> 
> How can I access to full _metadata from the Track datacos class?

You can call self._track_metadata